### PR TITLE
Update 3-rd party components

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

I have merged 7 PRs (#484, #485, #486, #487, #488, #490, #491) into one excluding #489 one as it requires changes in our code and extensive testing. After accepting this request, I will close #484, #485, #486, #487, #488, #490, #491.

Closes #484 #485  #486 #487 #488 #490 #491
## Type of change

Please delete options that are not relevant.

- [x] Code cleanup/refactoring

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,11 +6,11 @@ updates:
     directory: "/"
     # Check for updates once a day
```
Comment (t25kim) [inline .github/dependabot.yml:7]: Please correct the comments.
```suggestion
    # Check for updates once a month
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -6,11 +6,11 @@ updates:
...
     schedule:
-      interval: "daily"
+      interval: "monthly"
 # Enable version updates for Actions
   - package-ecosystem: "github-actions"
     # Look for `.github/workflows` in the `root` directory
     directory: "/"
     # Check for updates once a day
```
Comment (t25kim) [inline .github/dependabot.yml:14]:
```suggestion
    # Check for updates once a month
```

Comment (tdrozdovsky) [reply .github/dependabot.yml]: Done! Thank you!

File: .github/dependabot.yml (hunk 1 above)
Comment (tdrozdovsky) [reply .github/dependabot.yml]: Done!

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

**Review by suresh-lc — APPROVED** [review]
LGTM

**Review by MoonkiHong — APPROVED** [review]
Verified. Good to be merged.

**Pull request merged**